cadquery/cq.py

```diff
@@ -1346,6 +1346,48 @@ def move(self, xDist=0, yDist=0):
         newCenter = p + Vector(xDist, yDist, 0)
         return self.newObject([self.plane.toWorldCoords(newCenter)])
 
+    def slot(self, length, diameter, angle):
```

Comment:
One last technical proposal: could you default angle to zero? I think it will be natural for the users and will allow for nicer syntax when we implement something like `polarArray`

`def slot(self, length, diameter, angle=0):`

Reply:
I like that. That would be a good addition.